app/main/views/g_cloud.py

```diff
@@ -276,6 +276,25 @@ def search_services():
     )
 
 
+@direct_award.route('', methods=['GET'])
+def saved_search_overview():
+    projects = data_api_client.find_direct_award_projects(current_user.id).get('projects', [])
```

Comment:
Can we call it `buyer_dashboard` instead? The name you have here could too easily apply to just one saved search.